tests/models/test_aggregate.py

```diff
@@ -516,8 +517,13 @@ def progress_fn(p):
                                              nd_max_bins=div, seed=1)
         assert messages[0].progress == 0
         assert messages[0].status == "running"
+        assert messages[0].message == "Preparing"
+        assert messages[-2].progress < 1.0
```

Comment:
@st-pasha for a running task, can progress be equal to 1.0 or will always be < 1.0?

Reply:
It can. In fact, it must always reach 1.0 before switching to "finished", so you're quite likely to see progress message (1.0, "running") right before (1.0, "finished")

Reply:
Yeah, ok, let me make it `<= 1.0` then.